tests/installation/system_role/validate_default_role_transactional_server.pm

```diff
@@ -20,16 +20,12 @@
 use strict;
 use warnings;
 use base 'y2_installbase';
-use scheduler 'get_test_suite_data';
-use testapi;
-use Test::Assert 'assert_equals';
+use Test::Assert 'assert_matches';
 
 sub run {
-    my $default    = get_test_suite_data()->{system_role}{default};
-    my $controller = $testapi::distri->get_system_role_controller();
-
-    my $selected = $controller->get_selected_role();
-    assert_equals($default, $selected, 'Wrong System Role is pre-selected');
+    my $system_role = $testapi::distri->get_system_role_controller();
+    my $selected = $system_role->get_selected_role();
+    assert_matches(qr/Transactional/, $selected, 'Wrong System Role is pre-selected');
```

Comment:
Instead of writing here literal 'Transactional', we have this information stored at the page level, so we can just access the page and access its properties, it is the exception where we access from test layer to page layer, but makes it more consistent, so both string are in sync.

Reply:
I don't understand what you are suggesting here ?